packages/server/src/connector/index.ts

```diff
@@ -266,6 +310,39 @@ async function subscribeHeaders() {
   return
 }
 
+async function analyzeExtrinsics(blockHash: string) {
+  let signedBlock = await api.rpc.chain.getBlock(blockHash)
+  signedBlock.block.extrinsics.forEach(extrinsic => {
+    let methodName = extrinsic.method.methodName
+    let signer = extrinsic.signer
+    let args = extrinsic.method.args
+
+    if (methodName === 'nominate') {
+      api.createType('Vec<Address>', args[0]).forEach(arg => {
+        if (arg.toString() === settings.get().validatorId) {
+          notifications.send(
```

Comment:
make alias for these in notifications i.e. sendNominatedMessage(args) and extract it with get***Message to notifications/index.js or completely new file that will handle this to declutter connector